app/src/main/java/org/simple/clinic/deeplink/DeepLinkUpdate.kt

```diff
@@ -34,14 +34,22 @@ class DeepLinkUpdate : Update<DeepLinkModel, DeepLinkEvent, DeepLinkEffect> {
   }
 
   private fun patientFetched(
-      event: PatientFetched
+      event: PatientFetched,
+      model: DeepLinkModel
   ): Next<DeepLinkModel, DeepLinkEffect> {
-    val patient = event.patient
-    val effect = if (patient != null) {
-      NavigateToPatientSummary(patient.uuid)
+    val effect = if (event.patient != null) {
+      navigateToPatientSummary(model)
     } else {
       ShowPatientDoesNotExist
     }
     return dispatch(effect)
   }
+
+  private fun navigateToPatientSummary(model: DeepLinkModel): DeepLinkEffect {
+    return if (model.isLogTeleconsultDeepLink) {
```

Comment:
Why would we want to remove `isLogTeleconsultDeepLink` later? We still would want to check if navigating to `NavigateToPatientSummaryWithTeleconsultLog` or `NavigateToPatientSummary`, right ?

Reply:
WDYM? We will continue using it 🤔

Reply:
Oops, my mistake, there would PatientSummaryScreen with Teleconsult log only.